app/component/TimeInput.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component, PropTypes } from 'react';
+
+import debounce from 'lodash/debounce';
+
+const parseValue = (x) => {
+  const value = x.replace(/[^0-9:]/g, '');
```

Comment:
In Finnish the hour and minute should be separated by a full stop, rather than a colon, which we should support [Ref](http://www.kielitoimistonohjepankki.fi/haku/kellonaika/ohje/51)